db/migrate/20220515110719_add_devise_to_users.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
```

Comment:
There is a migration error so I was not able to run the project. Kindly fix the issue.
![devise](https://user-images.githubusercontent.com/35504149/168569425-07a219a4-bd0a-4d58-9d6e-4883d5800889.png)